<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>871</width>
      <height>571</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>871</width>
      <height>571</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>2D Scan</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>851</width>
        <height>531</height>
       </rect>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>851</width>
         <height>531</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="image_viewer">
            <property name="title">
             <string>Image Viewer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="4,4,4,1,1,1">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Scan Settings</string>
            </property>
            <widget class="QWidget" name="gridLayoutWidget">
             <property name="geometry">
              <rect>
               <x>33</x>
               <y>20</y>
               <width>361</width>
               <height>81</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0">
              <item row="0" column="2">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>End</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Step</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="LE_x_start"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Î¼m</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Y</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLineEdit" name="LE_y_stop"/>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="LE_y_start"/>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="LE_x_stop"/>
              </item>
              <item row="2" column="3">
               <widget class="QLineEdit" name="LE_y_step"/>
              </item>
              <item row="1" column="3">
               <widget class="QLineEdit" name="LE_x_step"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="formLayoutWidget_2">
             <property name="geometry">
              <rect>
               <x>29</x>
               <y>105</y>
               <width>371</width>
               <height>22</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>PMT Exposure Time (ms)</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="LE_pmt_exposure_time_in_ms"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Scan Progress</string>
            </property>
            <widget class="QWidget" name="formLayoutWidget">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>20</y>
               <width>361</width>
               <height>81</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <property name="labelAlignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="formAlignment">
               <set>Qt::AlignCenter</set>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Latest Count</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="LBL_latest_count">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Points Done</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="LBL_points_done">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Total Points</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="LBL_total_points">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Save File</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="LBL_save_file">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="BTN_select_save_file">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>110</y>
               <width>381</width>
               <height>23</height>
              </rect>
             </property>
             <property name="text">
              <string>Change Save File</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Plot Settings</string>
            </property>
            <widget class="QWidget" name="verticalLayoutWidget_3">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>30</y>
               <width>361</width>
               <height>81</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QFormLayout" name="formLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Min</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Max</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="plot_min"/>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="plot_max"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="CB_auto_minmax">
                <property name="text">
                 <string>Auto MinMax</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BTN_start_scanning">
            <property name="text">
             <string>Start Scanning</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BTN_stop_scanning">
            <property name="text">
             <string>Stop Scanning (Release FPGA)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BTN_go_to_max">
            <property name="text">
             <string>Go to Max</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>PMT Count Viewer</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
